testing/environments/snapshot-oss.yml

```diff
@@ -0,0 +1,35 @@
+# This should start the environment with the latest snapshots.
+
+version: '2.1'
+services:
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.0.0-alpha1-SNAPSHOT
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:9200"]
+      retries: 300
+      interval: 1s
+    environment:
+    - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+    - "network.host="
+    - "transport.host=127.0.0.1"
+    - "http.host=0.0.0.0"
+    - "xpack.security.enabled=false"
+
+  logstash:
```

Comment:
For some reasons this does not seem to work. When I start this alone with `docker-compose -f snapshot-oss.yml -f local.yml run logstash` the container starts. But when I run `ENV=snapshot-oss.yml make logs logstash` I just get an error for an unhealthy container and nothing in the logs. I tried to remove the volumes but no effect.

Reply:
solve it, problem was the ES container and not LS because of the x-pack licence part.